doc/criteria.md

```diff
@@ -396,7 +396,7 @@ There is an implied criterion that we should mention here:
+<li><a name="crypto_working"></a>The default security mechanisms within the software produced by the project MUST NOT depend on broken cryptographic algorithms (e.g., MD4, MD5, single DES, RC4, Dual_EC_DRBG), or use cipher modes that are inappropriate to the context, unless they are necessary to implement an interoperable protocol (where the protocol implemented is the most recent version of that standard broadly supported by the network ecosystem, that ecosystem requires the use of such an algorithm or mode, and that ecosystem does not offer any more secure alternative). The documentation MUST describe any relevant security risks and any known mitigations if these broken algorithms or modes are necessary for an interoperable protocol. (N/A allowed.) <sup>[<a href="#crypto_working">crypto_working</a>]</sup><dl><dt><i>Details</i>:<dt> <dd>ECB mode is almost never appropriate because it reveals identical blocks within the ciphertext as demonstrated by the <a href="https://blog.filippo.io/the-ecb-penguin/">ECB penguin</a>, and CTR mode is often inappropriate because it does not perform authentication and causes duplicates if the input state is repeated.  In many cases it's best to choose a block cipher algorithm mode designed to combine secrecy and authentication, e.g., Galois/Counter Mode (GCM) and EAX. Projects MAY allow users to enable broken mechanisms (e.g., during configuration) where necessary for compatibility, but then users know they're doing it.</dd><dt><i>Rationale</i>:<dt> <dd>If a cryptographic algorithm or mode is completely broken, then it cannot provide a useful cryptographic service. This is different from having a weakness; many cryptographic algorithms have some weaknesses, yet for backwards-compatibility it may sometimes be appropriate to use the algorithm anyway. "EAX" appears to be a name, not an abbreviation. The paper describing EAX, <a href="http://csrc.nist.gov/groups/ST/toolkit/BCM/documents/proposedmodes/eax/eax-spec.pdf">"A Conventional Authenticated-Encryption Mode" by M. Bellare, P.  Rogaway D.  Wagner (April 13, 2003)</a>, does not give an expansion.
```

Comment:
This is odd. How do we get a change to criteria.md without a change in the related locale English text? Did we not run this & check it in earlier?

Reply:
do you mean the same typo (`abbrevation`) in https://github.com/coreinfrastructure/best-practices-badge/pull/1958/files#diff-d75990e9242508478fd4e36275bb266bdbb9d38606deb0224b37ed82dd18a612 ?

Reply:
Oh THAT'S the change! Ah! Okay, that's fine.